HW8/task3.py

```diff
@@ -0,0 +1,12 @@
+def string_elem_to_up(elem):
+    print(f'Argument of function: {elem}')
+    if type(elem) == str:
```

Comment:
nit: nice function for checking type is `isinstance()`

Reply:
i am changed this mistake